test/parallel/test-child-process-fork-regr-gh-2847.js

```diff
@@ -42,5 +44,26 @@ var server = net.createServer(function(s) {
 
   // Send 2 handles to make `process.disconnect()` wait
   send();
-  send();
+  var s = send(function(err) {
+    // Ignore errors when sending the second handle because the worker
+    // may already have exited.
+    if (err) {
+      assert.strictEqual(err.code,
```

Comment:
Nit: Might be good just to follow the format of the previous case of swallowing `ECONNREFUSED` in the file: https://github.com/santigimeno/node/blob/regre-gh-2847/test/parallel/test-child-process-fork-regr-gh-2847.js#L19-L23